main.cc

```diff
@@ -1946,6 +1946,13 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 ss.local().uninit_address_map().get();
             });
 
+            // Need to make sure storage service does not use group0 before running group0_service.abort()
```

Comment:
No. I mean that we need to stop using group0 before executing `group0_service.abort()`

Reply:
Ok, then please clarify the comment. Because in fact we do want storage_service to use group 0 before we start shutdown so in particular before it is aborted .

Reply:
I changed "does not use" to "stopped using" if this is what confuses you.